be/src/vec/columns/column_array.cpp

```diff
@@ -422,7 +422,8 @@ void ColumnArray::reserve(size_t n) {
 
 //please check you real need size in data column, because it's maybe need greater size when data is string column
 void ColumnArray::resize(size_t n) {
```

Comment:
Is there the same problem for ColumnStruct?

Reply:
struct has no offsets array so do not has this problem